apps/Steam/install-32

```diff
@@ -21,7 +21,7 @@ fi
 
 
 echo "Installing steam.deb"
-install_package https://steamcdn-a.akamaihd.net/client/installer/steam.deb || error "Could not install steam deb"
+install_packages https://steamcdn-a.akamaihd.net/client/installer/steam.deb || error "Could not install steam deb"
```

Comment:
shouldn't this be merged into the install_packages used above @ https://github.com/Botspot/pi-apps/pull/653/commits/e44c16694666335259f199ff4f4efca7d91c22bd#diff-285f82f84b371da3c66033795fb753e8be3a88938eeeb5cfd4cd04ca72374f99R3